db/migrate/20170613144708_add_sti_to_container_image.rb

```diff
@@ -0,0 +1,20 @@
+class AddStiToContainerImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_images, :type, :string
+    add_column :container_images, :ems_ref, :string
+
+    say_with_time("Adding type and ems_ref columns to ContainerImage and setting type") do
+      ContainerImage.update_all(:type => "ManageIQ::Providers::Openshift::ContainerManager::ContainerImage")
```

Comment:
maybe `where.not.size(:size => nil)` ? (second command is for `where.size(:size => nil)`)